driver-core/src/main/com/mongodb/client/gridfs/model/GridFSDownloadByNameOptions.java

```diff
@@ -22,7 +22,9 @@
  * <p>Controls the selection of the revision to download</p>
  *
  * @since 3.1
+ * @deprecated use {@link GridFSDownloadOptions} instead.
  */
+@Deprecated
```

Comment:
I wasn't clear as to my intent here: what I meant is that we could do without the new GridFSDownloadOptions class entirely and continue to use GridFSDownloadByNameOptions even for the new overloads.